src/styles.scss

```diff
@@ -1111,10 +1111,15 @@ mat-icon.mat-icon-sm {
   }
 }
 
-// Style the float label of the mat-form-field
+// Purple outline and text for mat-form-field
+.mat-form-field-appearance-outline.mat-focused:not(.mat-form-field-invalid) .mat-form-field-outline,
+.mat-form-field.mat-focused:not(.mat-form-field-invalid) .mat-form-field-label,
```

Comment:
Maybe a copy-paste error, but there are duplicate selectors (lines 1116 and 1117, lines 1118 and 1119).

Reply:
oh thank you I didn't even notice